ios/RN/RNCamera.m

```diff
@@ -821,7 +836,8 @@ - (void)processVideoToAnimation:(NSURL *)outputFileURL {
     NSMutableArray *framesToKeep = [NSMutableArray arrayWithCapacity:framesNeeded];
     NSMutableArray *animatedFrames = [NSMutableArray arrayWithCapacity:framesNeeded * 2];
 
-    for (int i = 0; i < framesNeeded; i++) {
+    // Start int at 1 to skip first frame timestamped 0.00
+    for (int i = 1; i < framesNeeded; i++) {
```

Comment:
This will change the animation style from how it is currently. May not be a big deal since it's only dropping two frames. But I would be curious if there's another workaround here. If `0.00` poses a problem, can we just shift the first frame (or all the frames) ever so slightly to `0.01`?